***В университет на факультеты A и B поступило равное количество студентов, а на факультет C студентов поступило столько же, сколько на A и B вместе. Вероятность того, что студент факультета A сдаст первую сессию, равна 0.8. Для студента факультета B эта вероятность равна 0.7, а для студента факультета C - 0.9. Студент сдал первую сессию.***
---
- Какова вероятность, что он учится: 
- a) на факультете A 
- б) на факультете B 
- в) на факультете C?

H1 = студент был из A факультета

H2 = студент был из B факультета

H3 = студент был из C факультета

In [3]:
pA = 0.8
qA = 0.2 
pB = 0.7
qB = 0.3
pC = 0.9
qC = 0.1

H1 = pA * qB * qC
H2 = qA * pB * qC 
H3 = qA * qB * pC 

A = H1 / (H1 + H2 + H3)
B = H2 / (H1 + H2 + H3)
C = H3 / (H1 + H2 + H3)

A, B, C

(0.2608695652173913, 0.15217391304347824, 0.5869565217391305)

Ответ: Вероятность, что ученик, который сдал сессию был из:
- A факультета = 26%
- B факультета = 15.2%
- C факультета = 58.69%